renovate.json

```diff
@@ -42,6 +43,12 @@
       "matchPackageNames": ["openzaak/open-zaak"],
       "matchDatasources": ["docker"],
       "allowedVersions": "<= 1.15.0"
+    },
+    {
+      "matchPackageNames": ["infonl/zaakafhandelcomponent"],
```

Comment:
The matchPackageNames would need to be `ghcr.io/infonl/zaakafhandelcomponent` , as the package name is derived from the registry (with docker hub being the default). 
Maybe also add a filter for the digest type? Like
`"updateTypes": ["digest"],`

Reply:
clear. thanks! I updated it.